tesseract_collision/core/include/tesseract_collision/core/continuous_contact_manager.h

```diff
@@ -262,6 +262,12 @@ class ContinuousContactManager
    * @param type The type of contact test
    */
   virtual void contactTest(ContactResultMap& collisions, const ContactRequest& request) = 0;
+
+  /**
+   * @brief Applies settings in the config
+   * @param config Settings to be applies
+   */
+  virtual void applyCollisionCheckConfig(const CollisionCheckConfig& config);
```

Comment:
It think it may be best to avoid using the CollisionCheckConfig directly because it contains other information which is more to be used for executing a contact test. This could be confusing from a user perspective. What about changing it to something along the line of what is shown bellow?

``` c++
virtual void setIsContactAllowedFn(IsContactAllowedFn fn, ContactAllowedFnOverrideType override_type = ACMOverrideType::REPLACE);

virtual void setIsContactAllowedFn(tesseract_common::AllowedCollisionMatrix acm, ContactAllowedFnOverrideType override_type = ContactAllowedFnOverrideType::REPLACE);
```